src/arrayMethodSort.js

```diff
@@ -4,8 +4,27 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function(callback = (a, b) =>
+    a.toString() > b.toString()) {
```

Comment:
Better create function outside. It will be easier to read your code.

And you could also write an extended compareFunction like in [video](https://mate.academy/learn/javascript-advanced/array-iteration-methods#/video/152)